common/materialContainer/MaterialContainer.h

```diff
@@ -76,7 +76,7 @@ template <class T> class MaterialContainer
     , contentFreq(MC.contentFreq)
   {
   }
-  MaterialContainer& operator=(const MaterialContainer& mc) { return *this; }
+  MaterialContainer& operator=(const MaterialContainer&) { return *this; }
```

Comment:
Should we just get rid of this line? It seems like just returning `*this` without assigning the MaterialContainer's fields would be confusing. I guess we could also just assign all the fields if we wanted. Something like,
```c++
  MaterialContainer& operator=(const MaterialContainer& mc)
  {
    name = mc.name;
    contents = mc.contents;
    freq = mc.freq;
    smoothness = mc.smoothness;
    contentFreq = mc.contentFreq;
    return *this;
  }
```